scripts/build/setup/termux_setup_meson.sh

```diff
@@ -48,65 +54,65 @@ termux_setup_meson() {
 	fi
 
 	local CONTENT=""
-	echo "[binaries]" > $TERMUX_MESON_CROSSFILE
```

Comment:
Why this formating change here?

Reply:
That wasn't intentional. My editor auto-formatted on save. I'll revert it, if you wish so.

Reply:
Will make sense if excluded from this commit. So yes, please do it.

Reply:
Sure!